op-guide/offline-ansible-deployment.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Offline Ansible Deployment
+category: operations
+---
+
+# Offline Ansible Deployment
+
+## Prepare
+
+Before you start, make sure that you have:
+
+1. A Control Machine with the following requirements:
+
+    - Python 2.6 or Python 2.7
+    - Python Jinja2 2.7.2 and MarkupSafe 0.11 packages. You can use the following commands to install the packages:
+      
+      ```
+      pip install Jinja2==2.7.2 MarkupSafe==0.11
+      ```
+
+    - Access to the external network to install curl package and download binary.
+    - Access to the managed nodes via SSH using password login or SSH authorized_key login.
+
+2. Several managed nodes with the following requirements:
+
+    - 4 or more machines. At least 3 instances for TiKV. Don't deploy TiKV together with TiDB or PD on the same machine. See [deploying recommendations](https://github.com/pingcap/docs/blob/master/op-guide/recommendation.md).
+
+    - Recommended Operating system:
+
+      - CentOS 7.3 or later
+
+      - X86_64 architecture (AMD64)
+
+      - Kernel version 3.10 or later
+
+      - Ext4 file system
+
+    - The network between machines. Turn off the firewalls and iptables when deploying and turn them on after the deployment.
+
+    - The same time and time zone for all machines with the NTP service on to synchronize the correct time. If you are using the Ubuntu platform, install the ntpstat package.
+
+    - A remote user account which you can use to login from the Control Machine to connect to the managed nodes via SSH. It can be the root user or a user account with sudo privileges.
+
+    - Python 2.6 or Python 2.7
+
+    > **Note**: The Control Machine can be one of the managed nodes.
+
+## Install Ansible in the Control Machine
+
+1. Install Ansible offline on CentOS:
+
+  > Download the [Ansible](https://download.pingcap.org/ansible-2.3-rpms.el7.tar.gz) offline installation package to the Control Machine.
+  
+  ```ini
+  
+  tar -xzvf ansible-2.3-rpms.el7.tar.gz
+  
+  cd ansible-2.3-rpms.el7
+  
+  rpm -ivh PyYAML*.rpm libtomcrypt*.rpm libtommath*.rpm libyaml*.rpm python-
+  babel*.rpm python-backports*.rpm python-backports-ssl_match_hostname*.rpm
+  python-httplib2*.rpm python-jinja2*.rpm python-keyczar*.rpm python-
+  markupsafe*.rpm python-setuptools*.rpm python-six*.rpm python2-crypto*.rpm
+  python2-ecdsa*.rpm python2-paramiko*.rpm python2-pyasn1*.rpm sshpass*.rpm
+  rpm -ivh ansible-2.3.1.0-1.el7.noarch.rpm
+  ```
+
+2. After Ansible is installed, you can view the version using `ansible --version`.
```

Comment:
Great move! It's a good practice that you look into the code snip and document the steps not according to the Chinese version. @lilin90